src/util/util.c

```diff
@@ -726,7 +727,9 @@ void git__qsort_r(
 	qsort_r(els, nel, elsize, &glue, git__qsort_r_glue_cmp);
 #elif defined(GIT_QSORT_R_GNU)
```

Comment:
The code uses glue when dealing with traditional BSD qsort_r.  When both are present, it seems that GIT_QSORT_R_GNU should be preferred.

Reply:
They won't both be present, either the old variant or the new variant will be detected at configure time, but not both. Still, putting the most preferred variants at the top is less ambiguous, I guess.